src/AdditionalDocumentReference.php

```diff
@@ -47,6 +48,24 @@ public function setDocumentType(string $documentType): AdditionalDocumentReferen
         return $this;
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentDescription(): ?string
+    {
+        return $this->documentType;
```

Comment:
Shouldn't this be `return $this->documentDescription;`?

Reply:
Absolutely, fixed it. I admit I did not add any automated tests or anything... I don't use any of the getters myself.

(once all done, I can clean up the commits a bit to make them more compact)